examples/aws-data-lifecycle-manager.nix

```diff
@@ -0,0 +1,18 @@
+{ region ? "us-east-1"
+, accessKeyId ? "testing"
+, ...
+}:
+{
```

Comment:
Please set the network attribute in the example file

network.description = "AWS DLM Testing";

Otherwise the evaluation would fail

```
error: attribute 'network' missing, at /home/tewfikghariani/.cache/pypoetry/virtualenvs/nixops-aws-oxCLMhSo-py3.8/lib/python3.8/site-packages/nix/eval-machine-info.nix:51:26
warning: evaluation of the deployment specification failed; status info may be incorrect
```